README.win32.adoc

```diff
@@ -36,373 +36,292 @@ Mingw-w64 have no such restrictions.
-    echo INCLUDE="%INCLUDE%" >>C:\cygwin\tmp\msenv
-    echo FLPATH="`cygpath '%PFPATH%\flexdll'`" >>C:\cygwin\tmp\msenv
-    echo PATH="$VCPATH:$FLPATH:$PATH" >>C:\cygwin\tmp\msenv
-    echo export PATH LIB LIBPATH INCLUDE >>C:\cygwin\tmp\msenv
-    echo export OCAMLBUILD_FIND=/usr/bin/find >>C:\cygwin\tmp\msenv
-
-. Open a Cygwin shell and enter the following commands:
-
-    tr -d '\r' </tmp/msenv >.msenv64
-    echo '. $HOME/.msenv64' >>.bashrc
-
-Now, close the Command Prompt and the shell and you're set up for
-using the MS tools under Cygwin.
-
-
-To recompile OCaml, start a new Cygwin shell and change to the top-level
-directory of the OCaml distribution.  Then, do
-
-        cp config/m-nt.h config/m.h
-        cp config/s-nt.h config/s.h
-        cp config/Makefile.msvc64 config/Makefile
-
-Then, edit config/Makefile as needed, following the comments in this file.
-Normally, the only variable that need to be changed is
-
-        PREFIX      where to install everything
-
-Finally, use `make -f Makefile.nt` to build the system, e.g.
-
-        make -f Makefile.nt world
-        make -f Makefile.nt bootstrap
-        make -f Makefile.nt opt
-        make -f Makefile.nt opt.opt
-        make -f Makefile.nt install
-
-
-=== NOTES
-
-* Libraries available in this port: `num`, `str`, `threads`, `graphics`,
-  and large parts of `unix`.
-
-* The replay debugger is partially supported (no reverse execution).
-
-* The graphical browser ocamlbrowser is not supported.
+[[seflexdll]]
+== FlexDLL
+Although the core of FlexDLL is necessarily written in C, the `flexlink` program
+is, naturally, written in OCaml.  This creates a circular dependency if you wish
+to build entirely from sources.  Since OCaml 4.03 and FlexDLL 0.35, it is now
+possible to bootstrap the two programs simultaneously.  The process is identical
+for both ports.  If you choose to compile this way, it is not necessary to
+install FlexDLL separately -- indeed, if you do install FlexDLL separately, you
+may need to be careful to ensure that `ocamlopt` picks up the correct `flexlink`
+in your `PATH`.
+
+You must place the FlexDLL sources for Version 0.35 or later in the directory
+`flexdll/` at the top-level directory of the directory of the OCaml
+distribution.  This can be done in one of three ways:
+
+ * Extracting the sources from a tarball from
+   http://alain.frisch.fr/flexdll.html#download
+ * Cloning the git repository by running:
++
+  git clone https://github.com/alainfrisch/flexdll.git
+
+ * If you are compiling from a git clone of the OCaml repository, instead of
+   using a sources tarball, you can run:
++
+  git submodule update --init
+
+OCaml is then compiled as normal for the port you require, except that before
+compiling `world`, you must compile `flexdll`, i.e.:
+
+  make -f Makefile.nt flexdll world [bootstrap] opt opt.opt install
+
+ * `make -f Makefile.nt` will install FlexDLL by placing `flexlink.exe` (and the
```

Comment:
Oops - actually it should read `make -f Makefile.nt install` - it's describing the fact that a bootstrapped compilation also installs FlexDLL as part of OCaml.